### ISSUE TITLE
remove tabs from credits

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/rutherford-scattering#32]
In rutherford-scattering-main.js:

```js
team: 'Wendy Adams, Mike Dubson, Noah Finkelstein, \n\tDanielle Harlow, Ariel Paul, Kathy Perkins, \n\tNoah Podolefsky, Amy Rouinfar, Carl Weiman'
```

Credits are rendered using scenery-phet.MultiLineText, which only handles newline (\n).   So the tabs (\t) should be removed. You can also remove the spaces preceding the newlines.

So you'll have:

```js
team: 'Wendy Adams, Mike Dubson, Noah Finkelstein,\nDanielle Harlow, Ariel Paul, Kathy Perkins,\nNoah Podolefsky, Amy Rouinfar, Carl Weiman'
```

**Comment by schmitzware** [comment]
fixed in 9b7b76b3906e7f625d12566c7da2a16de2593d51

**Comment by pixelzoom** [comment]
I additionally removed unnecessary spaces. Closing.
